app/redux/selectors/SetsSelectors.js

```diff
@@ -128,6 +128,18 @@ export const getIsWorkingSet = (state, setID) => {
     return setID === currentSet.setID;
 };
 
+export const getSetIDs = (state) => {
```

Comment:
I'm not sure what the point of this function is. It literally creates a dictionary with keys as setIDs and values as false. That's not getting, that's creating something new.